recipes/meshio/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "meshio" %}
+{% set version = "3.2.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 925b10a27ad2eafcc731bb808625504c1c9d0a4e1fda19d2acbdfd1f39b4a290
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - meshio-convert = meshio._cli:convert
+    - meshio-info = meshio._cli:info
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools
+  run:
+    - python >=3
+    - numpy
+
+test:
+  imports:
+    - meshio
+  commands:
+    - meshio-convert --help
+    - meshio-info --help
+
+about:
+  home: "https://github.com/nschloe/meshio"
+  license: MIT
+  license_family: MIT
+  summary: "I/O for various mesh formats"
+  description: "I/O for various mesh formats"
```

Comment:
If you are not going to add more detail here, this key can be removed. No worries if you don't want to.